hist/hist/inc/TF1Convolution.h

```diff
@@ -35,6 +35,7 @@ class TF1Convolution : public TF1AbsComposition {
    Int_t    fNofPoints;                  ///< Number of point for FFT array
    Bool_t   fFlagFFT;                    ///< Choose FFT or numerical convolution
    Bool_t fFlagGraph = false;            ///<! Tells if the graph is already done or not
+   static Double_t fgExtraRangeFraction; ///<! Additional default fraction of the range used for FFT convolution
```

Comment:
Is there a reason for this not to be `static constexpr`? That would be convenient because it can be inlined then:
```C++
static constexpr Double_t fgExtraRangeFraction = 0.0;
```
Maybe that is better to make sure these liker errors such as reported in https://github.com/root-project/root/pull/8437 cannot happen.

Reply:
Ah nevermind, I see it is changed by `SetDefaultExtraRange`!